### ISSUE TITLE
When selecting OOIs on two pages, only the last page is taken into account

### ISSUE THREAD

**Issue by Donnype** [minvws/nl-kat-coordination#2174]

**Comment by underdarknl** [comment]
This is how Restfull applications work. You click to the next page, anything you selected on the current page is ignored.
We could make the pagination smarter and 'collect' all chosen ooi id's along the way to avoid this.

**Comment by noamblitz** [comment]
For now, we decided that the default filter would be OOIs with a declared scan level higher than zero to make it easier to select the relevant OOIs (in a short term).

**Comment by Donnype** [comment]
@underdarknl I don't believe this is how restful applications necessarily work, because this does not has to be solved in server side state. We actually do that anyways: moving back to the select ooi step after report generation also shows pre-selected oois fetched from the url.

**Comment by NoemiOverdijk** [comment]
We should consider showing the amount of selected OOIs as well.

**Comment by Rieven** [comment]
The title of this issue is different from the description. We can split this issue to only show
- Pre-selection of OOIs based on L1, L2, L3, L4 + declared
